taxi/tests/test_views.py

```diff
@@ -0,0 +1,98 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+CAR_URL = reverse("taxi:car-list")
+DRIVER_URL = reverse("taxi:driver-list")
+
+
+class PublicManufacturerTest(TestCase):
+    def test_manufacturer_login_required(self):
+        response = self.client.get(MANUFACTURER_URL)
+        self.assertNotEqual(response.status_code, 200)

```

Comment:
Better to check if it's equal specific status code in this case. != 200 means there could be any other status code, which is not really covers the case